translate/storage/pypo.py

```diff
@@ -27,6 +27,7 @@
 import re
 import six
 import textwrap
+import logging
```

Comment:
This import must be put in the right place. See comment about imports being alphabetically ordered.